.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request. The three fields below are mandatory. -->
+
+## Summary
+
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
+
+## Changelog
+
+<!-- Help reviewers and the release process by writing your own changelog entry. For an example, see:
+https://reactnative.dev/contributing/changelogs-in-pull-requests -->
+
+[CATEGORY] [TYPE] - Message
```

Comment:
This field is so necessary! But, I think it would be good to be included in ```Summary``` field. - ```What type of PR is this?```